json/Side_Text.txt

```diff
@@ -9164,1039 +9164,1039 @@
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "あ、あの……<%br>物かげに、隠れてました。",
-		"tr_text": "",
+		"tr_text": "Um...<%br>I was hiding... in the shadows.",
 		"fileID": 11
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "ええっ、どうしてだよ！？",
-		"tr_text": "",
+		"tr_text": "Why the heck were ya doin' that?!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "11",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "す、すみません……<%br>不安だったものですから。",
-		"tr_text": "",
+		"tr_text": "I-I'm sorry...<%br>I was nervous...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "はじめて会う人と話すのが……<%br>苦手なもので……ごめんなさい。",
-		"tr_text": "",
+		"tr_text": "When I'm meeting people for the first time...<%br>I get scared... Sorry.",
 		"fileID": 11
 	},
 	{
 		"eventNo": "13",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "そっか……まあいいや。<%br>オレはモア。よろしくな！",
-		"tr_text": "",
+		"tr_text": "I getcha, don't worry about it.<%br>I'm More. Nice to meet ya!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "14",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "わたしはジェネです。<%br>よろしくお願いしますね！",
-		"tr_text": "",
+		"tr_text": "My name is Gene.<%br>It's a pleasure to meet you!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "15",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "は、はい……",
-		"tr_text": "",
+		"tr_text": "R-right...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "16",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "……",
-		"tr_text": "",
+		"tr_text": "......",
 		"fileID": 11
 	},
 	{
 		"eventNo": "17",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "あ、ご、ごめんなさい……<%br>緊張して、うまくあいさつできなくて……",
-		"tr_text": "",
+		"tr_text": "Oh, s-sorry...<%br>I was so nervous, I forgot to introduce myself...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "18",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "大丈夫ですよ！<%br>少しずつ、仲良くなりましょう！",
-		"tr_text": "",
+		"tr_text": "That's not a problem!<%br>We'll just get to know each other slowly!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "19",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "そーかもしんねーけど……<%br>こんな調子で、大丈夫かな？",
-		"tr_text": "",
+		"tr_text": "OK, but...<%br>Are you sure this isn't gonna be a problem?",
 		"fileID": 11
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "任務完了、お疲れ様でした！",
-		"tr_text": "",
+		"tr_text": "Mission complete!<%br>Thank you for your hard work, everyone!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "は、はい……<%br>お疲れ様でした……",
-		"tr_text": "",
+		"tr_text": "O-okay...<%br>Thanks...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "オマエ、なかなかいい腕してるじゃんかっ！<%br>見直したぜ！",
-		"tr_text": "",
+		"tr_text": "Hey, you're pretty good!<%br>I might have to rethink my opinion of ya!",
```

Comment:
You could rephrase More's dialog to sound more like himself if you went with  `I might hafta rethink my opinion of ya` instead of `I might have to rethink my opinion of ya!`.